python/lib/dcoscli/tests/integrations/test_package.py

```diff
@@ -124,27 +124,33 @@ def test_repo_remove_multi_and_empty():
 
 def test_describe_nonexistent():
     stderr = (
-        b"Package [xyzzy] not found. Find the correct package name using "
-        b"'dcos package search'\n"
+        b"Error: package [xyzzy] not found. Find the correct package name using "
+        b"'dcos package search': 400 Bad Request\n"
     )
     assert_command(['dcos', 'package', 'describe', 'xyzzy'],
                    stderr=stderr,
                    returncode=1)
 
 
 def test_describe_nonexistent_version():
-    stderr = b'Version [a.b.c] of package [helloworld] not found\n'
+    stderr = b'Error: version [a.b.c] of package [helloworld] not found: 400 Bad Request\n'
     assert_command(['dcos', 'package', 'describe', 'helloworld',
                     '--package-version=a.b.c'],
                    stderr=stderr,
                    returncode=1)
 
 
 def test_describe_cli():
```

Comment:
You guys are probably getting tired of me going on about this, but I still think we should use table driven tests a bit more frequently. The following four tests would be a good use case for table driven tests. 

Ref: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

Reply:
+1 I think we can do it once we move tests to go. I'd like to not do it in python

Reply:
For the record this can be easily done in python: https://docs.pytest.org/en/latest/parametrize.html

Reply:
Done